src/test/java/com/wikia/webdriver/testcases/adstests/TestNoAdsForUsers.java

```diff
@@ -0,0 +1,26 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.geoedge.CountryCode;
+import com.wikia.webdriver.common.core.geoedge.GeoEdgeBrowserMobProxy;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.geoedge.GeoEdgeBrowserMobProxy'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)